docroot/themes/custom/mass_theme/templates/content/node--campaign-landing.html.twig

```diff
@@ -8,6 +8,12 @@
 {% extends 'node.html.twig' %}
 
 {% block preContent %}
+  {% if translationLinks %}
+    {% set translationLinks = translationLinks|merge({'background': "alternative"}) %}
```

Comment:
```suggestion
    {% set translationLinks = translationLinks|merge({'background': "alternative"}) %}
```
this will need to be updated based on mf comment

Reply:
Changed.

Reply:
Changed